app/src/lib/parser/token.js

```diff
@@ -116,20 +100,20 @@ export function set_token_concept(token, concept) {
 
 	const { stem, sense } = split_stem_and_sense(concept)
 	token.lookup_terms = [concept]
-	token.lookup_results = token.lookup_results.filter(result => result.stem === stem && result.sense === sense)
+	token.lookup_results = token.lookup_results.filter(result => result.stem === stem && result.concept !== null && result.concept.sense === sense)
```

Comment:
```suggestion
	token.lookup_results = token.lookup_results.filter(result => result.stem === stem &&  result?.concept.sense === sense)
```

I think this should give you the same result, would be worth double-checking a test for it.